training-node/app/services/tokenManager.js

```diff
@@ -1,10 +1,16 @@
 const jwt = require('jwt-simple'),
-  config = require('../../config');
+  config = require('../../config'),
```

Comment:
if you just use config.common.session you can do:
`session = require('../../config').common.session`

Reply:
Done !